javascript/features/gangs/gang.js

```diff
@@ -122,6 +141,21 @@ class Gang {
             setPlayerGangColor(player, color);
         }
     }
+
+    // Updates the skin of this gang.
+    updateSkinId(skinId) {
+        this.skinId_ = skinId;
+        for (const [player, _] of this.members_.entries()) {
+            if (player.isDisconnecting())
+                continue;
+
+            if (!this.usesGangSkin(player))
+                continue;
+
+            // Set the skin of all members. Will be done upon next spawn to avoid abuse.
+            pawnInvoke('OnSetPlayerSkinId', 'iii', player.id, skinId, 1);
```

Comment:
Should we gate this on `if (!server.isTest())`, to make sure that this is safe to call in tests?

Reply:
Why would we need to do that?

I've seen that the tests run a `MockPawnInvoke` where `pawnInvoke` only stores the calls into an array.

Reply:
Fair enough :)